domain/src/main/java/org/oppia/android/domain/state/StateDeck.kt

```diff
@@ -26,6 +28,8 @@ internal class StateDeck internal constructor(
   private val hintList: MutableList<Hint> = ArrayList()
   private lateinit var solution: Solution
   private var stateIndex: Int = 0
+  private var revealedHintIndex: Int = -1
```

Comment:
Maybe add the same comment for why the hint index is -1 over here

Reply:
done